controllers/index.js

```diff
@@ -1,9 +1,9 @@
 // Import just the router express
 const router = require('express').Router();
 // Import the index.js from 'api' folder
-//const apiRoutes = require('./api');
-const htmlRoutes = require ("./htmlRoutes");
-router.use ("/",htmlRoutes);
+const apiRoutes = require('./api');
+const homeRoutes = require ("./homeRoutes");
+router.use ("/", homeRoutes);
 // When a request is made to the /api route, it will be directed to the index.js in the 'api' folder.
 router.use('/api', apiRoutes);
```

Comment:
did this work for you? i had to swap 6 and 8 when i was testing for user profile because route 6 matches would end up matching all the other routes ... so i was never hitting route 8